<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Update</class>
 <widget class="QDialog" name="Update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>1011</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Update
{
	background:#2D2D2D;
}

</string>
  </property>
  <widget class="QLabel" name="upconnection">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upconnection
{
	background:white;
	color:black;
	border:none;
	padding-left:6px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic; text-decoration: underline; color:#00aaff;&quot;&gt;UPDATE INFO&lt;/span&gt;&lt;span style=&quot; font-style:italic; color:#00aaff;&quot;&gt; : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>650</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Back
{
	background:#00855C;
	border:none;
	color:white;
}

#Back:hover
{
	background:#2D2D2D;
	border:2px solid #00855C;
	color:#00855C;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#UpButton
{
	background:#EC3245;
	border:none;
	color:white;
}

#UpButton:hover
{
	background:#2d2d2d;
	border:2px solid #EC3245;
	color:#EC3245;
}</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QTableView" name="UptableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>511</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="UpName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Upid">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Updep">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="UpPhone">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Upgen">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="UpSalary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label
{
	color:white;
	background:#11110E;
padding-left:10px;
}</string>
   </property>
   <property name="text">
    <string>                                                         EMPLOYEE DATA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>89</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_2
{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>23</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3
{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>  ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>95</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_4
{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Department</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_5
{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Phone No</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_6
{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Joining Salary</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_7
{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>  Date Of Birth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_8
{
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ref">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>650</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Ref
{
	background:#00855C;
	border:none;
	color:white;
}

#Ref:hover
{
	background:#2D2D2D;
	border:2px solid #00855C;
	color:#00855C;
}</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
